src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -1003,4 +1013,8 @@
   </xsl:attribute-set>
   -->
 
+  <!-- Provide enough space for two digit labels (needed in bibliography). -->
```

Comment:
I submitted a patch to add a switch to fopub that enables hyphenation. See https://github.com/asciidoctor/asciidoctor-fopub/pull/63